spring-cloud-huawei-servicecomb-discovery/src/test/java/com/huaweicloud/servicecomb/discovery/client/ServiceCombClientTest.java

```diff
@@ -39,6 +39,7 @@ public void getServiceCenterInstances( @Injectable
     final int expectedCode = 200;
     Response response = new Response();
     response.setStatusCode(expectedCode);
+    url = "http://127.0.0.1:30100";
```

Comment:
```
@Injectable
      String url, @Injectable	      String url
```
This parameter can be deleted or declared as a constant.

Reply:
done